### PR TITLE
Updating the readme

### PR DESCRIPTION
**Pull request by hemajv**
adding a `getting started` section into the README
(addressing #119)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,122 @@ To install directly from this branch:
...
+The `prometheus-api-client` library consists of multiple modules which assist in connecting to a Prometheus host, fetching the required metrics and performing various aggregation operations on the time series data.
+
+#### Connecting and Collecting Metrics from a Prometheus host
+The `PrometheusConnect` module of the library can be used to connect to a Prometheus host. This module is essentially a class created for the collection of metrics from a Prometheus host. It stores the following connection parameters:
+
+-   **url** - (str) url for the prometheus host
+-   **headers** – (dict) A dictionary of http headers to be used to communicate with the host. Example: {“Authorization”: “bearer my_oauth_token_to_the_host”}
+-   **disable_ssl** – (bool) If set to True, will disable ssl certificate verification for the http requests made to the prometheus host
+
+```ruby
```
Comment (4n4nd) [inline README.md:35]: Can you change this to `python`?

Comment (4n4nd) [reply README.md:35]: @hemajv

File: README.md [hunk 2]
```diff
@@ -18,6 +18,122 @@ To install directly from this branch:
...
+prom.custom_query(query="prometheus_http_requests_total")
+
+# Now, lets try to fetch the `sum` of the metrics
+prom.custom_query(query="sum(prometheus_http_requests_total)")
+```
+
+We can also use custom queries for fetching the metric data in a specific time interval. For example, let's try to fetch the past 2 days of data for a particular metric in chunks of 1 day:
+
+```ruby
+start_time = parse_datetime("2d")
```
Comment (4n4nd) [inline README.md:60]: Can you show the imports for `parse_datetime`?

**Review by 4n4nd — CHANGES_REQUESTED** [review]
few minor changes, other than that LGTM :+1: Thanks!

File: README.md (hunk 1 above)
Comment (hemajv) [reply README.md]: my bad, fixed!

File: README.md (hunk 2 above)
Comment (4n4nd) [reply README.md:60]: also for `timedelta`

Comment (hemajv) [reply README.md:64]: added :+1:

**Review by 4n4nd — APPROVED** [review]

**Pull request merged**